packages/@o3r/analytics/tsconfig.build.json

```
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.build",
   "compilerOptions": {
+    "skipLibCheck": true, // Due to Jasmine / Jest fixture compiling conflicting on types
```

```suggestion
    "skipLibCheck": true, // Due to Jasmine / Jest fixture compilation conflicting on types
```
I'm not sure if this is a good suggestion? if you agree please apply to other files